docs/onboarding/PowerAuth-Enrollment-Onboarding-Server-2.0.0.md

```diff
@@ -18,6 +18,8 @@ For manual changes use SQL scripts:
 A new table `es_onboarding_process_configuration` has been added.
 Also added a foreign key `process_config_id` to the table `es_onboarding_process`.
 
+You have to insert at least one row into the table `es_onboarding_process_configuration`, and configure property `enrollment-server-onboarding.onboarding-process.default-type` (or ENV `ONBOARDING_PROCESS_DEFAULT_TYPE`) to work as a default process type.
```

Comment:
@michal-rozehnal-w When deploying, please remember this property and a database insert.